influxdb/defaults/main.yml

```diff
@@ -5,4 +5,6 @@ influxdb_data_dir: "/data"
 influxdb_index_version: "inmem"
 influxdb_wal_dir: "/var/lib/influxdb/wal"
 influxdb_retention_check_interval: "30m"
-influxdb_http_log_enabled: true
\ No newline at end of file
+influxdb_http_log_enabled: true
+influxdb_max_select_point: 10000
```

Comment:
These values should match influxdb default values and they should be overridden in our playbook / group_vars